modules/rosa-create-objects.adoc

```diff
@@ -0,0 +1,460 @@
+
+|--private
+|Restricts master API endpoint and application routes to direct, private connectivity.
+
+|--region
+|The AWS region (string) where your worker pool will be located. This argument overrides the `AWS_REGION` environment variable.
+
+|--service-cidr
+|Block of IP addresses (ipNet) for services. Example: `172.30.0.0/16`
+
+
+|--subnet-ids
+|The subnet IDs (string) to use when installing the cluster. Subnet IDs must be in pairs with one private subnet ID and one public subnet ID per availability zone. Subnets are comma-delimited. Example: `--subnet-ids=subnet-1,subnet-2`. Leave the value empty for installer-provisioned subnet IDs.
+
+|--version
+|The version (string) of OpenShift that will be used to install the cluster. Example: `4.3.10`
+
+|--watch
+|Watches the cluster installation logs.
+|===
+
+.Optional arguments inherited from parent commands
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--help
+|Shows help for this command.
+
+|--debug
+|Enables debug mode.
+
+|--interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|--v level
+|Log level for V logs.
+|===
+
+.Examples
+Create a cluster named `mycluster`.
+
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=mycluster
+----
+
+Create a cluster with a specific AWS region.
+
+[source,terminal]
+----
+$ rosa create cluster --cluster-name=mycluster --region=us-east-2
+----
+
+[id="rosa-create-idp_{context}"]
+== create idp
+
+Adds an identity provider (IDP) to define how users log in to a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create idp --cluster=<cluster_name> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--cluster
+|(Required) The name or ID (string) of the cluster to which the IDP will be added.
+
+|--ca
+|The path (string) to PEM-encoded certificate file to use when making requests to the server.
+
+|--client-id
+|The client ID (string) from the registered application.
+
+|--client-secret
+|The client secret (string) from the registered application.
+
+|--mapping-method
+|Specifies how new identities (string) are mapped to users when they log in. Default: `claim`
+
+|--name
+|The name (string) for the identity provider.
+
+|--type
+|The type (string) of identity provider. Options: `github`, `gitlab`, `google`, `ldap`, `openid`
+|===
+
+.GitHub arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--hostname
+|The optional domain (string) to use with a hosted instance of GitHub Enterprise.
+
+|--organizations
+|Specifies the organizations for login access. Only users that are members of at least one of the listed organizations (string) are allowed to log in.
+
+|--teams
+|Specifies the teams for login access. Only users that are members of at least one of the listed teams (string) are allowed to log in. The format is `<org>/<team>`.
+|===
+
+.GitLab arguments
+[cols="30,70"]
+|===
+|Option |Definition
```

Comment:
The header row (Option and Definition) are not being styled as a header. The only difference I see is that there isn't a line break after them in this table.

Reply:
Done. Looks fixed in my output.